javascript/javascript3/week2/warmup/script.js

```diff
@@ -0,0 +1,118 @@
+//Movies exercise
+//Create an array of bad movies
+const moviesUrl = "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+const myOl = document.createElement("ol");
+myOl.innerHTML = "Bad Movies";
+document.body.appendChild(myOl);
+
+const latestBadMv = document.createElement("ol");
+latestBadMv.innerHTML = "Bad Movies since 2000";
+document.body.appendChild(latestBadMv);
+fetch(moviesUrl)
+    .then(Response => Response.json())
+    .then(movies => {
+        const badMovies = movies.filter(movie => movie.rating < 4);
+        badMovies.forEach(mv => {
+            const myLi = document.createElement("li");
+            myLi.style.fontSize = "10px";
```

Comment:
I would avoid inline style and add a class instead `classList.add` and then customize more in main.css; but maybe for this exercise it is ok